client/redux/reducers/toggle-story-list.ts

```diff
@@ -0,0 +1,20 @@
+interface ToggleStoryListAction {
+  type: 'TOGGLE_STORY_LIST';
+  toggle: boolean;
+}
+
+const initialState = {
+  toggle: false,
+};
+
+export const toggleStory = (state = initialState, action: ToggleStoryListAction) => {
+  switch (action.type) {
+    case 'TOGGLE_STORY_LIST':
+      return {
+        ...state,
+        toggle: !state.toggle,
```

Comment:
Since `toggle` is a verb, it make the code a little confusing -- I'd go with something like `isVisible`

```typescript
    case 'TOGGLE_STORY_LIST':
      return {
        ...state,
        isVisible: !state.isVisible,
```